tf_agents/agents/ddpg/actor_network.py

```diff
@@ -61,10 +77,16 @@ def __init__(self,
         each item is a length-three tuple indicating (filters, kernel_size,
         stride).
       activation_fn: Activation function, e.g. tf.nn.relu, slim.leaky_relu, ...
+      kernel_initializer: Initializer to use for the kernels of the conv and
+        dense layers. If none is provided a default glorot_uniform
+      batch_squash: If True the outer_ranks of the observation are squashed into
```

Comment:
i'm not sure this arg should be exposed to the user.  why is it necessary?  the network will know if it's an RNN network or not, and can perform batch squash as necessary.